scripts/src/main/resources/scripts/command/docker

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Initiates docker setup
+function doSetup() {
+  if doIsBatch
```

Comment:
Here we should first check if docker command is already available in OS independent way (see other commandlets). If already installed, we are already done.

Reply:
Example:
https://github.com/devonfw/ide/blob/master/scripts/src/main/resources/scripts/command/ionic#L19